setup.py

```diff
@@ -6,13 +8,32 @@ def readme():
         return f.read()
 
 
+def _is_old_setuptool():
```

Comment:
Just noticed, it is actually `setuptools` (plural)

Reply:
In case you change it, `_is_outdated_setuptools()` is probably a little bit more correct.